### ISSUE TITLE
pygr.dbfile.WrongFormatError: db type could not be determined

### ISSUE THREAD

**Issue by atly2000** [cjlee112/pygr#6]
I am running a script that uses pygr. I got this error. What does it mean and what should I change?
```
Traceback (most recent call last):
  File "pwmScanning.py", line 239, in <module>
    genome = read_genome("/u/home/a/annly200/project-vitamind/REF/Homo_sapiens.GRCh37.75.dna.primary_assembly.fa")
  File "pwmScanning.py", line 45, in read_genome
    genome = seqdb.SequenceFileDB(fn)
  File "/u/home/a/annly200/.conda/envs/pygr_conda/lib/python2.7/site-packages/pygr/seqdb.py", line   424, in __init__
    seqLenDict = classutil.open_shelve(fullpath, 'r')
  File "/u/home/a/annly200/.conda/envs/pygr_conda/lib/python2.7/site-packages/pygr/classutil.py",  line 457, in open_shelve
    return dbfile.shelve_open(filename, flag=mode, useHash=useHash)
  File "/u/home/a/annly200/.conda/envs/pygr_conda/lib/python2.7/site-packages/pygr/dbfile.py", line 167, in shelve_open
    d = open_index(filename, flag, useHash, mode) # construct Shelf only if OK
  File "/u/home/a/annly200/.conda/envs/pygr_conda/lib/python2.7/site-packages/pygr/dbfile.py", line 106, in open_index
    d = open_anydbm(filename, flag)
  File "/u/home/a/annly200/.conda/envs/pygr_conda/lib/python2.7/site-packages/pygr/dbfile.py", line 46, in open_anydbm
    raise WrongFormatError(msg)
pygr.dbfile.WrongFormatError: db type could not be determined
```

**Comment by cjlee112** [comment]
TL;dr It looks like the Python anydbm package is trying to open a file Homo_sapiens.GRCh37.75.dna.primary_assembly.fa.seqlen and for some reason failing.  The simplest solution is to delete that file, which will force Pygr to rebuild it automatically.

Explanation: Pygr's SequenceFileDB keeps a shelve file to store the lengths of each of the sequences.  This file is just named by appending the suffix ".seqlen" to the sequence file name.  SequenceFileDB first tries to open that filename (in case it was already generated in a previous run) and if that raises NoSuchFileError then it creates and builds that shelve file.
